paket.dependencies

```diff
@@ -12,11 +15,15 @@ nuget System.Security.Cryptography.ProtectedData >= 4.4
 nuget NETStandard.Library ~> 2.0
 nuget Microsoft.NETCore.App ~> 2.1
 nuget System.Net.Http.WinHttpHandler >= 4.5
+nuget Microsoft.NETFramework.ReferenceAssemblies.net461
+nuget Nuget.Packaging
```

Comment:
I had to add these explicitly to make the netcore3.1 build of paket atually invoke correctly. otherwise the integration tests would blow up at runtime saying these were missing from the runtime.deps.json.